controllers/controller_ref_manager.go

```diff
@@ -210,56 +210,6 @@ func (m *EtcdDruidRefManager) FetchStatefulSet(ctx context.Context, etcd *druidv
 	return statefulSets, err
 }
 
-// ClaimStatefulsets tries to take ownership of a list of Statefulsets.
-//
-// It will reconcile the following:
-//   * Adopt orphans if the selector matches.
-//   * Release owned objects if the selector no longer matches.
-//   * Remove ownerReferences from the statefulsets and use annotations
-//
-// Optional: If one or more filters are specified, a Statefulset will only be claimed if
-// all filters return true.
-//
-// A non-nil error is returned if some form of reconciliation was attempted and
-// failed. Usually, controllers should try again later in case reconciliation
-// is still needed.
-//
-// If the error is nil, either the reconciliation succeeded, or no
-// reconciliation was necessary. The list of statefulsets that you now own is returned.
-func (m *EtcdDruidRefManager) ClaimStatefulsets(ctx context.Context, statefulSetList *appsv1.StatefulSetList, filters ...func(*appsv1.StatefulSet) bool) ([]*appsv1.StatefulSet, error) {
```

Comment:
Why is this no longer needed? Can you explain?

Reply:
Yes, we agreed a longer time ago that we're going to drop the claim functionality to simplify the entire reconciliation logic and handling.